Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs

```diff
@@ -78,20 +92,65 @@ private bool TryUseFood(IEntity user, IEntity target)
 
             var trueTarget = target ?? user;
 
-            if (trueTarget.TryGetComponent(out StomachComponent stomachComponent))
+            if (!trueTarget.TryGetComponent(out StomachComponent stomach))
             {
-                var transferAmount = ReagentUnit.Min(_transferAmount, _contents.CurrentVolume);
-                var split = _contents.SplitSolution(transferAmount);
-                if (stomachComponent.TryTransferSolution(split))
+                return false;
+            }
+
+            var utensils = utensilUsed != null
+                ? new List<UtensilComponent> {utensilUsed}
+                : null;
+
+            if (_utensilsNeeded != UtensilKind.None)
+            {
+                utensils = new List<UtensilComponent>();
+                var kinds = UtensilKind.None;
+
+                if (user.TryGetComponent(out HandsComponent hands))
```

Comment:
This check here would mean that if you were to have spaghetti and fork in your hand, you would just be able to click on the spaghetti instead of having to use the fork on the spaghetti.
I think it's more in the space station spirit to force users to click with the fork.

Reply:
That's just my opinion, though

Reply:
Changed it, directly using food no longer works if it has utensil requirements. Also removed eating food when clicking on nothing with it, following how drinks work.